relay-general/src/store/normalize.rs

```diff
@@ -1600,3 +1608,31 @@ fn test_past_timestamp() {
     }
     "###);
 }
+
+#[test]
+fn test_normalize_dist_none() {
+    let mut dist = None;
+    normalize_dist(&mut dist);
+    assert_eq!(dist, None);
+}
+
+#[test]
+fn test_normalize_dist_empty() {
+    let mut dist = Option::Some("".to_owned());
```

Comment:
my prev code suggestion goes for all testcases here fwiw

Reply:
Right, I did the lazy thing and pressed "Commit suggestion". Will change other instances as well.